commcare_connect/reports/views.py

```diff
@@ -253,46 +253,61 @@ class Meta:
         unknown_field_behavior = django_filters.UnknownFieldBehavior.IGNORE
 
 
-class NonModelFilterView(FilterView):
+class NonModelTableBaseView(FilterView):
+    # Inherit this for a tabular report
+    page_template = "reports/report_table.html"
+    htmx_table_template = "reports/htmx_table.html"
+    report_title = "Override this"
```

Comment:
Why is that better than `None`? `None` seems pretty clear and clean, and is nice convention to have, rather than a random value, even one that would be obvious on second look.